Sources/MockoloFramework/Utils/StringExtensions.swift

```diff
@@ -70,6 +78,23 @@ extension String {
     var isThrowsOrRethrows: Bool {
         return self == .throws || self == .rethrows
     }
+
+    var isSwiftKeyword: Bool {
```

Comment:
Instead of this, create var safeName: String { .. } in which it checks to see if self is contained in keywords and then return `self` if so.  Also move the keyword array out of StringExtension and make it a private (or fileprivate) constant. 

Then you can use safeName in the closureTemplate above.